### DISCUSSION TITLE
How to install on Proxmox LXC

### DISCUSSION THREAD

**Discussion by Worldrazor** [seerr-team/seerr#388] [Q&A] [answered]
Hello! As the title suggests I'm trying to install on Proxmox in a LXC container. When I'm trying to run the yarn commands it dosen't work. I assume it's because I have to write something different, but I really don't know what. 

For example: 
```
root@jellyseerr:~# yarn run build
00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'run'
```

I'm using Ubuntu, but could also change to debian if that's easier. 

Node.js version: v18.7.0

**Comment by fallenbagel** [comment]
Its because that's the wrong `yarn`. Do this. `apt purge yarn` and then do `npm install -g yarn`.

The yarn provided by the apt package manager is cmdlet (some other package)

**Reply by Worldrazor** [reply]
Thank you - that got me a little further. After I did that and then `yarn install` I got this error: 

```
root@jellyseerr:~/jellyseerr# yarn install
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
error preview-email@3.0.7: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.12"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Is my node.js to old? It shouldn't be

**Reply by Worldrazor** [reply, marked as answer]
I got it to work - thanks for the help! This program/service whatever you call it is super cool. Really nice work!

So first of all you have to do a `npm install -g yarn` to get the correct version of yarn. Then you have to use these commands to get the newest version of node.js

```
curl -fsSL https://deb.nodesource.com/setup_20.x | bash - &&\
apt-get install -y nodejs
```

**Reply by fallenbagel** [reply]
Glad you got it working! 🥳
(NVM also works if you want to install multiple versions of nodejs c:)

**Comment by Stooovie** [comment]
Can you guys help? I am also installing in a proxmox LXC (as I can not get Radarr to connect no matter what, so I'm installing everything in one LXC) and on "yarn start" I'm getting

`root@swizzin:~/jellyseerr# yarn start
yarn run v1.22.19
$ NODE_ENV=production node dist/index.js
node:internal/modules/cjs/loader:1073
  throw err;
  ^

Error: Cannot find module '/root/jellyseerr/dist/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1070:15)
    at Module._load (node:internal/modules/cjs/loader:923:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.2.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

Any tips? Thanks!

**Comment by Deekerman** [comment]
This site has a great collection of things. Jellyseerr is one.

@Stooovie Radarr too

[Proxmox VE Helper Scripts](https://tteck.github.io/Proxmox/)
